build/cli/src/arg/release.rs

```diff
@@ -25,6 +37,10 @@ pub enum Action {
     CreateDraft,
     /// Build the runtime image and push it to ECR.
     DeployRuntime(DeployRuntime),
+    /// Build the polyglot Ydoc image and push it to ECR.
+    DeployYdocPolyglot(DeployYdocPolyglot),
```

Comment:
should really be called `BuildAndDeploy...` but I understand this is consistent with the previous entry

Reply:
This is because the constructor automagically creates the `run release deploy-ydoc-polyglot` CLI argument. Otherwise, the command will look like `run release build-and-deploy-ydoc-polyglot`